openstack/blockstorage/extensions/quotasets/requests.go

```diff
@@ -15,3 +15,9 @@ func GetDefaults(client *gophercloud.ServiceClient, projectID string) (r GetResu
 	_, r.Err = client.Get(getDefaultsURL(client, projectID), &r.Body, nil)
 	return
 }
+
+// GetDetail returns detailed public data about a previously created QuotaSet.
+func GetDetail(client *gophercloud.ServiceClient, projectID string) (r GetDetailResult) {
```

Comment:
I think we should call this `GetUsage`. I understand that this API call returns more than what's "in use" and so "detail" certainly makes sense. But the word "detail" isn't found in the API docs or in the cinder code when referencing this call. However, the API docs, URL, and code all mention "usage".

As well, if this is the only API call that returns what's in use, then someone searching for how to return the quota usage will more easily discover this.

Please let me know if you feel differently, though.

Reply:
My only concern about this is the lack of consistency with [the compute quotaset](https://github.com/gophercloud/gophercloud/blob/e7e2296ede45795fce803ac2906a442acb16f1c1/openstack/compute/v2/extensions/quotasets/requests.go#L15), which uses `Detail` rather than `Usage`. That being said, I don't feel too attached and will update per your comments; if you do decide consistency is a virtue we can discard those changes before merge.